test/Microsoft.Azure.Devices.Edge.Test/Metrics.cs

```diff
@@ -123,7 +123,7 @@ static class EdgeConfigBuilderEx
         public static void AddTemporaryModule(this EdgeConfigBuilder builder)
         {
             const string Name = "stopMe";
-            const string Image = "mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0";
```

Comment:
I'm realizing...this is not a default that can be overridden. So this will cause the ValidateMetrics test to fail until 1.4 is actually released. Let's change this to 1.1 for now.